components/squad/squad_custom.lua

```diff
@@ -24,21 +26,24 @@ function CustomSquad.run(frame)
 			})
 			:name({name = player.name})
 			:role({role = player.role})
-			:date(player.joindate, 'Join Date:&nbsp;')
+			:date(player.joindate, 'Join Date:&nbsp;', 'joindate')
 
 		if squad.type == Squad.TYPE_FORMER then
-			row:date(player.leavedate, 'Leave Date:&nbsp;')
+			row:date(player.leavedate, 'Leave Date:&nbsp;', 'leavedate')
 			row:newteam({
 				newteam = player.newteam,
 				newteamrole = player.newteamrole,
 				newteamdate = player.newteamdate,
 				leavedate = player.leavedate
 			})
 		elseif squad.type == Squad.TYPE_INACTIVE then
-			row:date(player.inactivedate, 'Inactive Date:&nbsp;')
+			row:date(player.inactivedate, 'Inactive Date:&nbsp;', 'inactivedate')
 		end
 
-		squad:row(row:create())
+		squad:row(row:create(
+			Variables.varDefault('squad_name',
```

Comment:
where does that var come from?
i don't see it being set by any of the modules

Reply:
Only places across all wikis that define wiki-var`squad_name`:

https://liquipedia.net/heroes/index.php?title=User:Tephus&action=edit
https://liquipedia.net/starcraft2/Module:Squad
https://liquipedia.net/starcraft/Module:Squad

Reply:
and for sc/sc2 it is only to be passed along from the start to the rows